int16[11] angle # unit: [0.1 deg]
